etl/harmonize.py

```diff
@@ -14,7 +15,10 @@
 from owid.catalog import Dataset
 from rapidfuzz import process
 
-from etl.paths import REFERENCE_DATASET
+from etl.paths import REFERENCE_DATASET, STEP_DIR
+
+# Path to latest regions definitions file. We update aliases from harmonize
+REGION_DEFINITIONS_FILE = STEP_DIR / "data/garden/regions/2023-01-01/regions.yml"
```

Comment:
Instead of needing to update this every time we create a new version, you can ensure you always load the latest version:
```

REGION_DEFINITIONS_FILE = sorted((STEP_DIR / "data/garden/regions/").glob("*/regions.yml"))[-1]
```

Reply:
Sweet!